code/game/machinery/vending_types.dm

```diff
@@ -867,27 +867,104 @@
 		/obj/item/plantspray/pests = 20,
 		/obj/item/reagent_containers/syringe = 5,
 		/obj/item/storage/bag/plants = 5
-	)
+		)
```

Comment:
fix formatting on these closing parentheses - should be as it was before

how it is:
```
= list(
    /foo
    )
```

how it should be:
```
= list(
    /foo
)